src/main/java/edu/cornell/kfs/concur/rest/application/ConcurApplication.java

```diff
@@ -0,0 +1,31 @@
+package edu.cornell.kfs.concur.rest.application;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+import edu.cornell.kfs.concur.service.impl.ConcurWebServiceImpl;
+
+@ApplicationPath("/concur/v1.0")
```

Comment:
I think changing this to just v1 would be more consistent with the base financials APIs. (i.e. "/concur/v1")